### PR TITLE
LPS-143552 | Verify the Vanilla Counter remote app can be viewed in a portal instance

### PR DESCRIPTION
**Pull request by susanchen3**
**Background**
Add test coverage for [LPS-143439](https://issues.liferay.com/browse/LPS-143439)

**Test Plan**
- Create a virtual instance
- Assert there are no console errors
- Navigate to virtual instance
- Assert Vanilla Counter is present in remote apps portlet
- Navigate back to default instance 

**JIRA Ticket:**
https://issues.liferay.com/browse/LPS-143552

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase [hunk 1]
```diff
@@ -86,6 +88,37 @@ definition {
...
 
+	@description = "LPS-143552. Verify that remote app of type Custom Element can be displayed by portal instance"
+	@priority = "5"
+	test CustomElementCanBeDisplayedByInstance {
+		property portal.acceptance = "true";
+		property test.assert.warning.exceptions = "true";
+		property test.name.skip.portal.instance = "RemoteApps#CustomElementCanBeDisplayedByInstance";
+
+		task ("Create virtual instance") {
+			SAML.addVirtualInstances(virtualHosts = "www.able.com");
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase:99]: Use JSON instead for setup phase, there's HeadlessPortalInstanceAPI.addPortalInstance()

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase [hunk 2]
```diff
@@ -86,6 +88,37 @@ definition {
...
+		task ("Assert custom remote app is present") {
+			RemoteApps.goToRemoteAppsPortlet();
+
+			RemoteApps.viewTableEntries(
+				entryName = "Vanilla Counter",
+				entryType = "Custom Element");
+		}
+
+		task ("Navigate back to default instance") {
+			User.firstLoginPG();
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase:118]: Use Navigator.openURL(); instead, skip the sign in process and just navigate to the baseURL.

**Review by john-co — CHANGES_REQUESTED** [review]

**Comment by susanchen3** [comment]
ci:test:sf

**Comment by susanchen3** [comment]
sf fixed in [#111660](https://github.com/brianchandotcom/liferay-portal/pull/111660), isnt synced yet

**Review by john-co — APPROVED** [review]
Looks good!

**Comment by susanchen3** [comment]
ci:test:sf

**Comment by susanchen3** [comment]
ci:forward

**Pull request closed without merging**